test/geometry/transform/test_image_registrator.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import torch
+from torch.autograd import gradcheck
+
+import kornia
+import kornia.testing as utils  # test utils
+from kornia.geometry.transform import ImageRegistrator
+from kornia.testing import assert_close
+
+
+class TestSimilarity:
+    def test_smoke(self):
+        expected = torch.eye(3)[None]
+        for r, sc, sh in zip([True, False], [True, False], [True, False]):
+            s = kornia.geometry.transform.Similarity(r, sc, sh)
```

Comment:
```suggestion
            sim = kornia.geometry.transform.Similarity(r, sc, sh)
```
to avoid issues to navigate with debugger ;)